CHANGELOG.md

```diff
@@ -1,7 +1,46 @@
 # Changelog
-
 All notable changes to this project will be documented in this file.
 
+## 3.0.0 - 2024-05-02
+[Full Changelog](https://github.com/nextcloud-libraries/nextcloud-l10n/compare/v2.2.0...v3.0.0)
+
+### Breaking changes
+Instead of also sanitizing the replacing variables, now only the result is sanitized, see [pull request #648](https://github.com/nextcloud-libraries/nextcloud-l10n/pull/648).
+
+This will improve the performance if multiple placeholders are used and it allows this, 
+while the string is still sanitized:
+```js
+t(
+	`See {linkstart}documentation{linkend}`,
```

Comment:
```suggestion
	'See {linkstart}documentation{linkend}',
```
We should never use computing backticks, they just provoke the thinking you could use variables directly in there 🙈

Reply:
Copy past fail :facepalm: